docs/hive-integration/kube-api-getting-started.md

```diff
@@ -0,0 +1,493 @@
+# Kube API - Getting Started Guide
+
+This document is a step-by-step guide that demonstrates how to deploy a single-node cluster end to end.
+
+**Note**:
+* This document is not meant to expand on each and every resource in detail; [other documents already do that](README.md). Instead, expect the details needed to understand the context and what is currently happening.
+
+* The order in which actions are performed here is optional, and you may choose to go by a different one. The one order restriction the API does have is in regards to creating NMstateConfigs (which are optional) before InfraEnv, for more details, [check the warning mentioned here](README.md#NMStateConfig).
+## What To Expect
+* A walk through custom resource definitions (CRDs) and understand how they relate to each other.
+
+* A clear understanding of what happened, both in OpenShift / Kubernetes and assisted-service backend per each action described below.
+
+## How To Use This guide
+
+* Make sure that for each step you follow, you understand how and why it is done
+
+* You are encouraged to copy and paste the below-mentioned commands to your terminal and see things in action.
+
+* Note the inline comments in the `yaml` examples. There is no need to remove those in case you which to copy and create the resources directly in your environment.
+
+## Assumptions
+* You have deployed both the operator and assisted-service, [using these instructions](../operator.md).
```

Comment:
There are potentially multiple ways to deploy them. Even using OLM, you might point-and-click with the console or create OLM's Subscription and other resources directly. Or deploy as part of ACM. etc. I'd clarify here that following those exact instructions are just *a way* to deploy the operator. Something simple like "You have deployed ..., for example by using these instructions." could make it clear that there's nothing special in those instructions that are required for this documentation to be applicable.